docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -962,3 +965,56 @@ To remove a key, such as channels, and all of its values:
 To configure channels and their priority for a single
 environment, make a ``.condarc`` file in the :ref:`root directory
 of that environment <config-channels>`.
+
+.. _threads:
+
+Configuring number of threads
+=============================
+
+You can use your ``.condarc`` file or environment variables to
+add configuration to control the number of threads. You may
+want to do this to tweak conda to better utilize your system.
+If you have a very fast SSD, you might be able to shorten the
+time it takes for conda to create environments and
+install/remove packages.
+
+**repodata_threads**
+
+* Default: None
```

Comment:
I think instead of putting just `Default` here it would make more sense to put `Default number of threads` or something. For example, `Default: None` threads really means that the default number of threads is probably 1